### PR TITLE
make the bazel PR builder only build for now

### PR DESCRIPTION
**Pull request by mikedanese**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1119)

### CODE REVIEW

File: jobs/pull-kubernetes-bazel.sh [hunk 1]
```diff
@@ -17,7 +17,16 @@ set -o errexit
...
+TEST_CMD=(bazel test)
+TEST_OPTS=("--test_output=errors" "--test_tag_filters" "-skip")
+TEST_TARGETS=("//cmd/..." "//pkg/..." "//plugin/...")
+
+BUILD_TARGETS=("//cmd/..." "//pkg/..." "//federation/..." "//plugin/..." "//build-tools/..." "//test/...")
+BUILD_CMD=(bazel build)
+
+# only build for now.
+# TODO(mikedaanese): make this run tests
+"${BUILD_CMD[@]}" "${BUILD_TARGETS[@]}" && rc=$? || rc=$?
```
Comment (spxtr) [inline jobs/pull-kubernetes-bazel.sh]: `bazel build //cmd/... //pkg/... //federation/... //plugin/... //build-tools/... //test/... && rc=$? || rc=$?` and comment out the test line. No need for variables like `BUILD_CMD`.

Comment (mikedanese) [inline jobs/pull-kubernetes-bazel.sh]: Why not

Comment (spxtr) [inline jobs/pull-kubernetes-bazel.sh]: I don't think `"${BUILD_CMD[@]}" "${BUILD_TARGETS[@]}"` is easier to understand *or* change. Shell scripts should be dumb.

**Comment by mikedanese** [comment]
@spxtr hates self documenting code. Fixed.

**Pull request merged**